lib/hydra/derivatives/processors/jpeg2k_image.rb

```diff
@@ -73,20 +73,18 @@ def layer_rates(layer_count, compression_numerator)
         rates.map(&:to_s).join(',')
       end
 
-      protected
```

Comment:
I'm seeing that the following methods are now becoming public. What was your reason for removing the `protected` declaration?